src/contracts/UpgradeRegressionManager.sol

```diff
@@ -208,22 +201,22 @@ contract UpgradeRegressionManager is IUpgradeRegressionManager {
 
     _rollbackId = getRollbackId(_targets, _values, _calldatas, _description);
 
-    // Revert if the rollback not proposed or already queued.
-    if (rollbackQueueExpiresAt[_rollbackId] == 0) {
-      revert UpgradeRegressionManager__NotQueueable(_rollbackId);
-    }
+    Rollback storage _rollback = _rollbacks[_rollbackId];
```

Comment:
This should also *not* have a `_` prefix, because it touches a storage slot.

Reply:
Tracked at https://github.com/ScopeLift/governance-rollback/issues/53